docs/releases/v2018.1.1.rst

```diff
@@ -0,0 +1,50 @@
+Gluon 2018.1.1
+==============
+
+Bugfixes
+~~~~~~~~
+
+* Fix a bug leading to configuration loss on upgrade under certain circumstances
+  (`#1496 <https://github.com/freifunk-gluon/gluon/issues/1496>`_)
+
+  The issue can only occur when upgrading from 2018.1 and there are multiple
+  mirror entries in *site.conf* (specifically, an early failure for one of the
+  mirrors, e.g. during DNS resolution, followed by a successful upgrade from a
+  different mirror triggers the issue).
+
+  This is a regression is Gluon 2018.1.
+
+* Fix next-node ARP issue
+  (`#1488 <https://github.com/freifunk-gluon/gluon/issues/1488>`_)
+
+  A routing table issue led to ARP requests being sent from the next-node IPv4 address, but with
+  a node-specific source MAC address. This could make the next-node IPv4 address unreachable.
+
+  This is a regression is Gluon 2018.1.
+
+* Fix build on hosts with glibc 2.28
+
+  Fixed by various tool upgrades in LEDE (*bison*, *e2fsutils*, ...)
+
+Other changes
+~~~~~~~~~~~~~
+
+* Linux kernel has been updated to v4.4.148
+
+Known issues
+~~~~~~~~~~~~
+
+* Default TX power on many Ubiquiti devices is too high, correct offsets are unknown (`#94 <https://github.com/freifunk-gluon/gluon/issues/94>`_)
+
+  Reducing the TX power in the Advanced Settings is recommended.
+
+* The MAC address of the WAN interface is modified even when Mesh-on-WAN is disabled (`#496 <https://github.com/freifunk-gluon/gluon/issues/496>`_)
+
+  This may lead to issues in environments where a fixed MAC address is expected (like VMware when promicious mode is disallowed).
+
+* Inconsistent respondd API (`#522 <https://github.com/freifunk-gluon/gluon/issues/522>`_)
+
+  The current API is inconsistent and will be replaced eventually. The old API will still be supported for a while.
+
+* Frequent reboots due to out-of-memory on weak hardware in larger meshes
+  (`#1243 <https://github.com/freifunk-gluon/gluon/issues/1243>`_)
```

Comment:
do we not want to copy the known issue section from 2018.1? especially a sentence further describing the current status of the OOM stuff is missing here

Reply:
Fixed.